src/rules/selector-attribute-quotes/index.js

```diff
@@ -33,6 +33,8 @@ export default function (expectation) {
       parseSelector(rule.selector, result, rule, selectorTree => {
 
         selectorTree.walkAttributes(attributeNode => {
+          if (!attributeNode.operator) { return }
```

Comment:
@davidtheclark it is for performance, if we have `[src]` we return early, but we have `[src][title="value"]`, we should check each attribute.